libs/wlgen/wlgen/rta.py

```diff
@@ -101,6 +101,39 @@ def __init__(self,
         # Setup RTA callbacks
         self.setCallback('postrun', self.__postrun)
 
+    @staticmethod
+    def __setGovernorsPerformance(target):
+        if 'cpufreq' not in target.modules:
+            log = logging.getLogger('RTApp')
+            log.warning('cpufreq module not loaded,'
+                        ' skipping setting frequency to max')
+            return {}
+
```

Comment:
Since we are at that, there can be another condition to test: the performance governor should be available.
```python
if 'performance' not in target.cpufreq.list_governors(0):
   log.warning('performance cpufreq governor not available, '
               'skipping setting frequency to max')
   return {}

```